### ISSUE TITLE
Cannot download data - File size 3MB (881kb unzipped) ; Error message "The system is currently not bale to process your request. Try to refresh the page and please report the issue id it continues" ; I have tried refreshing page but to no avail

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4876]
**Cannot download data - File size 3MB (881kb unzipped) ; Error message &quot;The system is currently not bale to process your request. Try to refresh the page and please report the issue id it continues&quot; ; I have tried refreshing page but to no avail**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/e1f490b88c5314adf5272f7d5fdf1fd5:f914f42b3c5ebc59fb2b8e91bec6f36b9291df0aeea04142ace4e66d2d136a6691fc1d62d1ec97f16b5b608da3950bceab5d342bb8e041e294f0aa495455249aeb4a698192076fbfe69079ab772b35d31696a83830d352ae9ac4cad0524db1b0351456d9af861ae1233699664edeb08f7ca70e460db4b34642ed5b9d0b52abb019c3bc93931091919c1f00682e7383c78a895402a1baaa055ec0d6322d4aca93) - [Send email](https://www.gbif.org/api/feedback/user/mailto/e1f490b88c5314adf5272f7d5fdf1fd5:f914f42b3c5ebc59fb2b8e91bec6f36b9291df0aeea04142ace4e66d2d136a6691fc1d62d1ec97f16b5b608da3950bceab5d342bb8e041e294f0aa495455249aeb4a698192076fbfe69079ab772b35d31696a83830d352ae9ac4cad0524db1b0351456d9af861ae1233699664edeb08f7ca70e460db4b34642ed5b9d0b52abb019c3bc93931091919c1f00682e7383c78a895402a1baaa055ec0d6322d4aca93)
System: Chrome 115.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/download?gadm_gid=IND.3.5_1
Window size: width 1536 - height 786
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-07-25T05:22:29.641Z',to:'2023-07-25T05:28:29.641Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-07-25T05:22:29.641Z',to:'2023-07-25T05:28:29.641Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ktotsum** [comment]
The download link was created after this error message was shown on the screen (all systems are operational). This issue can be reproduced but it is not always the case. 
![image](https://github.com/gbif/portal-feedback/assets/138471170/dc4995fc-dd0f-4b5e-9695-8b47a5c8afc6)

**Comment by MattBlissett** [comment]
https://github.com/gbif/portal-feedback/issues/4878 and probably https://github.com/gbif/portal-feedback/issues/4875 are duplicates.

We've noticed the problem — the statistics added to the download information yesterday have caused some download API calls to take up to 10× longer, which adds up to several seconds in some cases. The portal appears to give up, perhaps at 15 seconds, and shows the error.

I've added a temporary fix, which means it's still a bit slow to create a download but not slow enough to cause an error (5-9s). Marcos is working on a permanent fix, which should bring us back to the original speed (1-4s to create a download).
